crux/src/Crux/SVCOMP/Witness.hs

```diff
@@ -0,0 +1,329 @@
+-- | This module defines a data type for SV-COMP witness automatons, as
+--   specified in https://github.com/sosy-lab/sv-witnesses.
+
+{-# Language NamedFieldPuns #-}
+module Crux.SVCOMP.Witness where
+
+import           Data.ByteString (ByteString)
+import qualified Data.ByteString.Char8 as B (unpack)
+import qualified Data.List as L (intercalate)
+import           Data.Maybe (catMaybes)
+import           Data.Time.Format.ISO8601 (iso8601Show)
```

Comment:
Note to self: this module was introduced in `time-1.9`, and unfortunately, GHC 8.6 is bundled with an earlier version of `time`, causing this to fail to build on GHC 8.6. I'll need to see if I can backport `iso8601Show` somehow.